### DISCUSSION TITLE
Embedding not loaded ( Beginner's Question )

### DISCUSSION THREAD

**Discussion by aiaisd** [TheLastBen/fast-stable-diffusion#2504] [Q&A]
I uploaded the embeddings to the embeddings folder in Google Drive, restarted Stable Diffusion, but the embeddings are not loaded, even after pressing the refresh button. However, I have multiple Colab accounts and for some reason only one of them loaded the embeddings.

How can embedding be loaded? By the way, I would like to load Easy Negative, etc.

I am a beginner who does not know any codes or commands. Please tell me in a simple way. Thanks very much for "TheLastBen/fast-stable-diffusion".

**Comment by OrganicBeej** [comment]
> I uploaded the embeddings to the embeddings folder in Google Drive, restarted Stable Diffusion, but the embeddings are not loaded, even after pressing the refresh button. However, I have multiple Colab accounts and for some reason only one of them loaded the embeddings.
> 
> How can embedding be loaded? By the way, I would like to load Easy Negative, etc.
> 
> I am a beginner who does not know any codes or commands. Please tell me in a simple way. Thanks very much for "TheLastBen/fast-stable-diffusion".

Hi, I'm no tech expert but I did suffer from this issue a few days ago if I had started Colab or a local install (I'm on an iMac) with an SDXL model loaded from previous working in A1111 (mine was an SDXL Model from Civitai.com)

All you have to do is select an older 1.5 model from your drop-down list (I changed the VAE too but prob don't need to?) and when the 1.5 model has loaded, click the red/pink box to show models/Loras/Embeddings etc and navigate to 'Embeddings' (I also had to do this with my Lora's directory too) and hit the 'refresh' icon to the right. Your Embeddings/Lora's etc should refresh in the list once you do this. You can then re-load any SDXL model and they shld stay there.

I don't know why this happens but it's not a Colab issue because this happened to me first on my local install.

Hope that works for you :) 👍🏻

**Comment by aiaisd** [comment]
Thanks for your comment! I tried it and when the initial model was v2.1-768, embedding was not installed. However, when the initial model was 1.5, embedding was installed.
